### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by zhangrui231** [SWHL/RapidVideOCR#44] [Q&A] [answered]
我测试了一下直接使用图片路径传递图片给RapidOcr如果图片两端空白太多，会识别成空。
如果用padding_img 处理一下图片用cv2.copyMakeBorder给图片添加相框，确实能识别出来但是有时候会把一行分成多段，没有直接传图片路径准确。

**Comment by zhangrui231** [comment]
![0_00_23_232__0_00_25_650_1000006231280005612800720](https://github.com/SWHL/RapidVideOCR/assets/20070528/e9f11a6a-ddc7-4d9b-87b3-512f0e765ad9)
这张图片是不用padding_img 处理会识别会空，处理后能正确识别
![0_00_03_003__0_00_06_548_1000006221280005712800720](https://github.com/SWHL/RapidVideOCR/assets/20070528/42f6f595-a454-4c4a-b3af-de14931a42c8)
这张图片不处理能识别正确，处理了就识别成多段打印的结果是这样
[[[[275.0, 272.0], [1412.0, 275.0], [1412.0, 416.0], [274.0, 413.0]], 'Hello world! 手', '0.8730344573656718'], [[[1342.0, 274.0], [1842.0, 269.0], [1843.0, 420.0], [1343.0, 424.0]], '我是 C', '0.7131001591682434'], [[[1801.0, 270.0], [2867.0, 280.0], [2866.0, 423.0], [1800.0, 413.0]], 'arrie Anne， 又', '0.8238681895392281'], [[[2801.0, 266.0], [4757.0, 266.0], [4757.0, 428.0], [2801.0, 428.0]], '欢迎收看 计算机科学速成课！', '0.8820517786911556']]

**Comment by SWHL** [comment, marked as answer]
这种图不用检测的，可以调节一下参数，这种直接走识别。效果就好了。
调节`width_height_ratio`，计算一下这张图的宽高比，将这个`width_height_ratio`值该为比图的宽高比大的值即可。

https://github.com/RapidAI/RapidOCR/blob/29d5f5fc01fbff7c49926a3c297fa8a3fb1624af/python/rapidocr_onnxruntime/config.yaml#L7
